src/gConsent/manage/revokeAccessGrant.ts

```diff
@@ -38,29 +39,27 @@ import { getBaseAccess } from "../util/getBaseAccessVerifiableCredential";
 const { quad, namedNode } = DataFactory;
 
 /**
- * Makes a request to the access server to revoke a given Verifiable Credential (VC).
- *
- * @param vc Either a VC, or a URL to a VC, to be revoked.
- * @param options Optional properties to customise the request behaviour.
- * @returns A void promise.
- * @since 0.4.0
+ * @internal
  */
-async function revokeAccessGrant(
+async function revokeAccessCredential(
   vc: DatasetWithId | VerifiableCredential | URL | UrlString,
   options: Omit<AccessBaseOptions, "accessEndpoint"> = {},
+  type: NamedNode<string>[] = [
```

Comment:
```suggestion
  types: NamedNode<string>[] = [
```
Arrays are typically named plural as opposed single values.